Stresser/EtwManager.h

```diff
@@ -25,11 +24,9 @@ const std::wstring LOGGER_NAME(KERNEL_LOGGER_NAME);
  * Define the default properties and the default logfile for the ETW session
  * and start the trace session, stop it, etc.
  */
-class EtwManager : public IEtwEventHandler, IEventCollector
+class EtwManager : public IEventCollector
 {
 public:
-	explicit EtwManager(std::vector<EtwEventTypes> eventTypes, std::function<void(PEVENT_RECORD)> callback);
```

Comment:
Add:
`EtwManager() = default;`

Reply:
Make sure to specify the `default`, and `delete` everywhere.
This is best practice.